src/pages/coding-challenges.astro

```diff
@@ -1,11 +1,12 @@
 ---
 import Layout from "../layouts/Layout.astro";
-import Logo from "../components/Logo.astro";
+// import Logo from "../components/Logo.astro";
```

Comment:
I think this one wants deleting 😁

Reply:
Happy to remove this.

Reply:
Awesome. I'll watch for the change.
If you have time, can you also just give the eslint and prettier scripts a kick and commit and push any changes they make?
```
npm run lint
npm run format
```
Maybe we can clean up some of these extra spaces and make the PR smaller.

Reply:
No rush on these by the way! I know it's getting late! 😬 😁